action/protocol/producer/donatemsg.go

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) 2019 IoTeX
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package producer
+
+import (
+	"math"
+	"math/big"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/pkg/errors"
+
+	"github.com/iotexproject/iotex-core/action"
+	"github.com/iotexproject/iotex-core/pkg/util/byteutil"
+	"github.com/iotexproject/iotex-core/proto"
+)
+
+var (
+	donateToProducerFundBaseGas    = uint64(10000)
```

Comment:
strongly feel "deposit" is better than "donate"

Reply:
ack